scripts/flag_validator.bash

```diff
@@ -4,9 +4,16 @@ SCRIPTS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 SWUPD_DIR="$SCRIPTS_DIR"/..
 SWUPD="$SWUPD_DIR"/swupd
 
-swupd_commands=(info autoupdate check-update update bundle-add bundle-remove bundle-list search-file diagnose repair os-install mirror clean hashdump)
+swupd_commands=(info autoupdate check-update update bundle-add bundle-remove
+	bundle-list search-file diagnose repair os-install mirror clean hashdump 3rd-party)
 conflict=0
 
+check_command_exist() {
+
+	local command=$1
+	sudo "$SWUPD" "$command" --help 2>&1 > /dev/null | grep "Error: unrecognized subcommand" > /dev/null
```

Comment:
this works kind of backwards of what I would expect. It is called check_command_exist but it returns False when it exists, and True when it doesn't.